content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -138,6 +138,19 @@ EOF
 
 ```
 
+## JsonPath Cheats with JQ
+
+Json output often creates complex nested structures which can be simplified with 'JQ' JSON processor
+
+```bash
+# Produce a period-delimited tree of all keys returned for nodes
+kubectl get nodes -o json | jq -c 'path(..)|[.[]|tostring]|join(".")'
+
```

Comment:
Hi @gkramer . Thanks for opening this pull request. A couple of questions:
Does this change fit in with the existing `jq` commands on the page?
Should there be a separate heading for `jq` related kubectl commands?

Reply:
JQ is a great tool for exploring the JSON output prior to building a jsonpath query, but probably doesnt require its own page as it's not as tightly integrated with kubectl as jsonpath support.

I've since merged these suggestions into one of the other headings.

Reply:
Thanks @gkramer .